backend/src/main/java/com/cic/service/person/PersonService.java

```diff
@@ -12,14 +12,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.web.multipart.MultipartFile;
+import com.cic.openapi.model.Gender;
 import com.cic.openapi.model.Person;
 
 public class PersonService {
   private static final Logger LOGGER = LoggerFactory.getLogger(PersonService.class);
-  private final Map<String, PersonKeys> menteeMapping;
+  private final Map<String, PersonKeys> mapping;
 
-  public PersonService(Map<String, PersonKeys> menteeMapping) {
-    this.menteeMapping = menteeMapping;
+  public PersonService(Map<String, PersonKeys> mapping) {
+    this.mapping = mapping;

```

Comment:
I would have thought keeping the name `menteeMapping` would be preferred since it is a more descriptive name than `mapping`

Reply:
menteeMapping only works if its only doing mentees but since this service is generic enough it can be used for both,
honestly debating making this a super and creating two separate subclasses but the only diff would be the mapping which makes it kind of silly